fala/templates/adviser/search.html

```diff
@@ -35,68 +31,100 @@ <h1 class="govuk-heading-xl"> Find a legal aid adviser or family mediator </h1>
+                  'text' : "For example, SW1H",
+                },
+                'attributes': {
+                   'maxLength': 30,
+                },
+                'id': 'id_postcode',
+                'classes': 'govuk-!-width-one-third',
+                'name': "postcode"
+              }) }}
+            {% endif %}
+            <p class="govuk-body">Or</p>
+            {% if 'name' in form.errors or form.errors['__all__'] %}
+              {{ govukInput({
+                'label': {
+                  'text': "Organisation name",
+                  'classes': 'govuk-label--s',
+                },
+                'attributes': {
+                   'maxLength': 100,
+                },
+                'value': form.name.value(),
+                'errorMessage': { 'text': errorMessage.err },
+                'id': 'name',
+                'classes': 'govuk-!-width-one-third',
+                'name': "name"
+              }) }}
+            {% else %}
+              {{ govukInput({
+                'label': {
+                  'text': "Organisation name",
+                  'classes': 'govuk-label--s',
+                },
+                'attributes': {
+                   'maxLength': 100,
+                },
+                'value': form.name.value(),
+                'id': 'name',
+                'classes': 'govuk-!-width-one-third',
+                'name': "name"
+              }) }}
+            {% endif %}
 
             <div class="form-group filters">
-              <div class="govuk-form-group">
-                <fieldset class="govuk-fieldset fala-tickbox-columns_new">
-                  <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-                    <label class="govuk-label govuk-label--s">
-                      Categories of law (optional)
-                    </label>
-                  </legend>
-                  <div class="govuk-checkboxes govuk-checkboxes--small">
-                    {% for value, label_text in form.categories.field.choices %}
-                      <div class="govuk-checkboxes__item">
-                        <input
-                          class="govuk-checkboxes__input"
-                          id="id_categories_{{ loop.index0 }}_{{ label_text }}"
-                          name="categories"
-                          type="checkbox"
-                          value="{{ value }}"
-                          {% if value in request.GET.getlist('categories','') %}
-                            checked
-                          {% endif %}
-                        >
-                        <label class="govuk-label govuk-checkboxes__label" for="id_categories_{{ loop.index0 }}_{{ label_text }}">
-                          {{ label_text }}
-                        </label>
-                      </div>
-                    {% endfor %}
-                  </div>
-                </fieldset>
-              </div>
+              {{ govukCheckboxes({
```

Comment:
Can somehow get this added back in to the checkboxes `id="id_categories_{{ loop.index0 }}_{{ label_text }}`. It make the Google Analytics easier to understand as the Id for the categories selected will then have a name rather than just "category 2"

Reply:
Do we need the loop index? It's now back as it was but w/o the loop index e.g. id_categories_Crime

Reply:
No the `loop index` just gave a number to the id, before the name was assigned to it. So it can be removed. Our analytics aren't dependant on it either.